packages/styles/scss/components/button/_tokens.scss

```diff
@@ -330,11 +330,11 @@ $button-disabled: (
     ),
     (
       theme: themes.$g90,
-      value: rgba(141, 141, 141, 0.3),
+      value: rgb(141 141 141 / 30%),
```

Comment:
Is the `rgb` approach preferred moving forward?? Pretty cool format 👀

Reply:
I had originally modified it to that format, but I was getting compilation errors when running the style tests. Any ideas on how to get around that?